### PR TITLE
Change rubocop auto-correct command

### LINKED ISSUE #18: Rubocop template may not be safe
## Summary
When installing the rubocop template, at the end it runs the `rubocop -A`. Is it safe if we install the template not in a new project?

## Suggestion
I want rubocop create a `.rubocop_todo.yml` file instead. Script can calculate count of files in project and run `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit {count_of_files_in_project}`. [Article](https://evilmartians.com/chronicles/rubocoping-with-legacy-bring-your-ruby-code-up-to-standard)

### PR DESCRIPTION
**Pull request by RasimKhusaenov**
## Summary
Resolves #18.

Now the rubocop template generates a todo file instead of automatically resolving all offenses.

## Test plan
1. Generate a new app / Select an existing one.
2. Apply `rubocop` template.
3. You should see `.rubocop-todo.yml` with all the offenses in your project.

### CODE REVIEW

File: templates/rubocop.rb [hunk 1]
```diff
@@ -48,4 +48,6 @@
 end
 
 run "bundle binstubs rubocop"
-run "bin/rubocop -A"
+
+count_of_files = Dir["**/*"].length
```
Comment (DmitryBarskov) [inline templates/rubocop.rb:52]: Did you mean files count?

Comment (DmitryBarskov) [inline templates/rubocop.rb:52]: Maybe we add only ruby files from `app`, `lib`, `spec`, `test` and `config` directories?

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: templates/rubocop.rb [hunk 2]
```diff
@@ -48,4 +48,6 @@
 end
 
 run "bundle binstubs rubocop"
-run "bin/rubocop -A"
+
+files_count = Dir["app/**/*", "lib/**/*", "spec/**/*", "test/**/*", "config/**/*"].length
```
Comment (DmitryBarskov) [inline templates/rubocop.rb:52]: What do you think about counting only .rb files? Also does rubocop scan .erb files?

Comment (RasimKhusaenov) [reply templates/rubocop.rb:52]: What gives us a point number of files? For the argument, the main thing is to specify a number that exceeds the number of files that will be checked.

**Review by DmitryBarskov — APPROVED** [review]

**Pull request merged**